docs/integrations-and-sdks/vapi.mdx

```diff
@@ -0,0 +1,62 @@
+---
+id: integrations-and-sdks-vapi
+description: Learn how to integrate Speechmatics STT with Vapi.
+---
+
+import CodeBlock from '@theme/CodeBlock';
+
+# Vapi
+
+Vapi offers native integration with Speechmatics for real-time, highly accurate speech-to-text in AI voice agents. This integration is designed to handle real-world conversation complexities like background noise, diverse accents, and multiple speakers. 
+
+Vapi is perfect for:
+
+- **Customer Service & Contact Centers**: Automating customer interactions with AI agents that can accurately understand and respond to diverse accents, background noise, and industry-specific jargon, leading to better customer experiences and call routing.
+- **Healthcare & Medical**: Building agents for tasks like appointment booking, patient triage, and clinical note-taking (ambient AI) where precise transcription of medical terminology is critical for compliance and efficiency.
+- **Multi-Lingual Communication**: Developing global voice agents that can operate in over 55 languages, automatically detecting the language spoken and providing accurate transcription and translation.
+- **Meeting & Interview Platforms**: Creating AI assistants that can accurately transcribe multi-speaker conversations in real-time and identify who said what, providing full context for downstream tools and automated summaries.
+- **Educational Tools**: Implementing interactive study partners that can quiz students, elaborate on topics, and provide immediate feedback through natural voice interactions. 
+
+## Features
+
+- **Native Availability**: Speechmatics is a natively available transcriber option within the Vapi platform, simplifying the setup process for developers.
+- **High Accuracy & Low Latency**: The integration provides an accurate and low-latency input layer, crucial for natural, real-time conversations with AI agents.
+- **Speaker Diarization**: Speechmatics is the only transcriber on Vapi that provides speaker diarization, which identifies and labels who said what in multi-speaker scenarios.
+- **Flexible Deployment**: Speechmatics STT is available in Vapi's managed platform.
+
+## Quickstart
+
+### Requirements
+- Vapi >= 1.2
+- Speechmatics Account (https://portal.speechmatics.com)
```

Comment:
Do I need a speechmatics account? It doesn't seem like I need it in the code example.